Routines/Camera.py

```diff
@@ -0,0 +1,15 @@
+import picamera
```

Comment:
I don't think we need this as a "routine". It'd be best if we can trigger a start and stop recording separately as we respond to alerts. The data file setup looks great though.

Reply:
I can't confirm that the threading is working. I get errors when I run it with the threading.Thread() in the code. I might be running it wrong